### DISCUSSION TITLE
installation

### DISCUSSION THREAD

**Discussion by ghost** [threeML/threeML#526] [Q&A]
Hello professor! I want to ask some questions about my threeML installation! I installed the Linux subsystem under Windows, but when I entered Jupyter lab in Terminal, it showed that the Python kernel did not have threeML kernel. Do you configure Jupyter Lab under (base) or where? I just want to ask you about the configuration after your Installation tutorial, thank very much!
![image](https://user-images.githubusercontent.com/89920593/147814872-5a76bb4f-cd05-4a14-ab11-59255c3b78e1.png)

![image](https://user-images.githubusercontent.com/89920593/147814891-3f7d888b-e721-4823-ba4c-1d0de8a32da3.png)

**Comment by grburgess** [comment]
one has to use the environment which 3ML is installed in.

**Reply by ghost** [reply]
Ok, thank you for your answer
